docker/muranoservice/README.md

```diff
@@ -0,0 +1,237 @@
+# Installing murano as service
+## Install murano packages
+
+Enable Openstack repositories:
+
+    apt install software-properties-common
+    add-apt-repository cloud-archive:newton
+
+Then, install murano packages:
+
+    apt-get install murano-api murano-engine python-muranoclient
+
+## Create database
+
+To create the database, complete these steps:
+
+Use the database access client to connect to the database server as the root user:
+
+    $ mysql -u root -p
+
+Create the murano database:
+
+    CREATE DATABASE murano;
+
+Grant proper access to the heat database:
```

Comment:
I just prefer:

Grant proper access to the heat database, replacing _MURANO_DBPASS_ with a suitable password: